warehouse/query-core/src/main/java/datawave/query/jexl/functions/GroupingRequiredFilterFunctions.java

```diff
@@ -5,6 +5,7 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.HashSet;
+import java.util.Iterator;
```

Comment:
As far as I can tell there are no functional changes in this class, just reorganization of the code.  Am I correct in this observation?  This is ok, just makes it harder to review.

Reply:
Yes, that was an unused import. Removed.